### DISCUSSION TITLE
Packages/com.unity.ugui/manual/script-HorizontalLayoutGroup

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#263] [Notes]
# Packages/com.unity.ugui/manual/script-HorizontalLayoutGroup

https://docs.unity3d.com/Packages/com.unity.ugui@latest/index.html?subfolder=/manual/script-HorizontalLayoutGroup.html

**Comment by radiatoryang** [comment]
For a (better) explanation of how Layout Groups work, see [this blog](https://www.hallgrimgames.com/blog/2018/10/16/unity-layout-groups-explained).
